data/rulei/rulei.txt

```diff
@@ -90,6 +89,24 @@ mission "Rulei Planet"
 			`	It's best you don't remain here for too long.`
 				decline
 
+
+mission "Umbral's Reach"
+	invisible
+	landing
+	to complete
+		never
+	to offer
+		has "First Contact: Rulei: offered"
+		has "visited system: L-118"
+	on enter "Kanguwa"
+		log "Factions" "Rulei" `The ancient Rulei ship stated to be careful of "Umbral's Reach".`
```

Comment:
I don't think this is worth logging. See the Archon nanobots logbook entry: it doesn't mention to avoid Sayaiban, but it instead logs the insight behind the Archon's actions.

Reply:
Good point.